homework_2/fizzbuzz.py

```diff
@@ -0,0 +1,19 @@
+# You should write a function that will receive a positive integer and return:
+# "Fizz Buzz" if the number is divisible by 3 and by 5;
+# "Fizz" if the number is divisible by 3;
+# "Buzz" if the number is divisible by 5;
+# The number as a string for other cases.
+
+
+def fizz_buzz(number):
+    if not isinstance(number, int) or number <= 0:
+        raise ValueError("Input must be a positive integer")
+
+    if number % 3 == 0 and number % 5 == 0:
+        return "Fizz Buzz"
+    elif number % 3 == 0:
+        return "Fizz"
+    elif number % 5 == 0:
+        return "Buzz"
+    else:
+        return str(number)
```

Comment:
I would suggest to avoid spreading return keywords around if else blocks

```
    if number % 3 == 0 and number % 5 == 0:
        res =  "Fizz Buzz"
    elif number % 3 == 0:
        res =  "Fizz"
    elif number % 5 == 0:
        res =  "Buzz"
    else:
        res =  str(number)
    
    return res
```